sail/README.md

```diff
@@ -0,0 +1,53 @@
+<!-- action-docs-header action="action.yml" -->
+
+<!-- action-docs-header action="action.yml" -->
+
+<!-- action-docs-description action="action.yml" -->
+## Description
+
+Runs the Ocean Sail command for a specific type of integration
```

Comment:
Added explenation because we dont have this section :<


![image](https://github.com/port-labs/ocean-github-action/assets/63305203/02e89e3d-dbce-4785-8f74-04e2625e846c)